src/System.Security.Cryptography.Algorithms/src/ApiCompatBaseline.net463.txt

```diff
@@ -3,6 +3,8 @@ MembersMustExist : Member 'System.Security.Cryptography.ECDsa.Create(System.Secu
 MembersMustExist : Member 'System.Security.Cryptography.ECDsa.Create(System.Security.Cryptography.ECParameters)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Security.Cryptography.ECDsa.ExportExplicitParameters(System.Boolean)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Security.Cryptography.ECDsa.ExportParameters(System.Boolean)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'System.Security.Cryptography.ECDsa.FromXmlString(System.String, System.Security.Cryptography.ECKeyXmlFormat)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'System.Security.Cryptography.ECDsa.ToXmlString(System.Security.Cryptography.ECKeyXmlFormat)' does not exist in the implementation but it does exist in the contract.
```

Comment:
So given that, the methods would only show up in the ECDsaCng and ECDsaOpenSsl meaning not present in ECDsa in Algorithms assembly. And no good way to add them if we want?